examples/NeuralNetwork-load/sketch.js

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) 2024 ml5
```

Comment:
Just remembering #76 and we should create a friendlier, less formal standard header for the examples, I don't think a copyright or license reference is needed or helpful, that can be established in this repo.

Reply:
Ah yes, I created this example by duplicating another example and modifying it and didn't pay attention to the header. Perhaps we can make a separate PR updating all the example headers?